### DISCUSSION TITLE
Could you add copy selection as well as option to strip off links to text?

### DISCUSSION THREAD

**Discussion by aravindk100** [mvdkwast/obsidian-copy-as-html#17] [Ideas]
First this is a great plugin. Lot simpler than the other flow I am using with Pandoc to export and import into Outlook. I have a couple of requests,
1) Copy selection as HTML – There is another plugin that does similar capability, but that plugin does not appear to be actively developed  and has a few bugs that limit the usage of it. With all the capability, this plugin as this would be an wonderful addition.
2) Can you also add an option to strip off links and convert them to plain text - Links are for other notes, which does not mean much when I copy and paste into another software like Outlook? 

Thanks for considering these.

**Comment by mvdkwast** [comment]
For reference, moved these requests to issues #19 and #18.
